docs/tutorials/projects/filtered-camera.md

```diff
@@ -181,7 +180,18 @@ Once your dataset is ready, train a new ML model on that dataset.
 
 For more information, see [Train a model](/ml/train-model/).
 
-### Upload an existing ML model
+### Use an existing ML model
+
+You can either [use an existing model from the registry](#select-an-existing-model-from-the-registry) or [upload an ML model](#upload-an-existing-ml-model) trained outside the Viam platform.
+
+#### Select an existing model from the Registry
+
+The Viam registry hosts trained ML models that users have made public, which you can use to deploy classifiers or detectors for your use case onto your robot instead of training your own.
+You can see all available ML models in the [Viam Registry](https://app.viam.com/registry).
+
+Once you've decided on a model to use, continue to [Add the ML model service](#add-the-ml-model-service).
+
+#### Upload an existing ML model
```

Comment:
Line 209 below should likely change similarly to something like:

> To upload and use an existing ML model:

Or similar, as "use existing" is now a discrete separate action that does not require uploading (the new MR ML model approach)

Reply:
done